LOTUS_RELEASE_FLOW.md

```diff
@@ -95,12 +95,29 @@ By policy, the team will usually wait until about 3 weeks after the final releas
 
 Unless a security issue is actively being exploited or a significant number of users are unable to update to the latest version, security fixes will not be backported to previous releases.
 
+## Branch and Tag Strategy
+* Releases branches have the name `release/vX.Y.Z`
+* By the end of the release process, a `release/vX.Y.Z` branch will have an associated `vX.Y.Z` tag and likely also `vX.Y.Z-rcN` tags.
+* `master` branch is almost always where `release/vX.Y.Z` branches are created from.  The exception is we need to do an emergency rushed patch release and can't risk pulling in other changes that have already landed in `master`.  In that case, `release/vX.Y.Z+1` will be created from `release/vX.Y.Z`.
```

Comment:
I think we have to think a little bit more about these lines, and how we can convey the branching strategy with the separation of `Node` and `Miner`. Ref: https://github.com/filecoin-project/lotus/pull/12096#issuecomment-2222938021

- `release/vX.Y.Z` branches for node releases
- `release/miner/vX.Y.Z` branches for miner releases
- By the end of the release process:
   - A `release/miner/vX.Y.Z` branch will have an associated `miner/vX.Y.Z` tag to it
   - A `release/vX.Y.Z` branch will have an associated `vX.Y.Z` tag to it.


Writing this in text can be a little bit confusing given the similar looking nature of these branches.

Reply:
Good point!  I like your expanded text.  We could also do colors to help differentiate?  https://stackoverflow.com/questions/35465557/how-to-apply-color-on-text-in-markdown. Feel free to adjust/merge or I can try this out when more at the keyboard a bit later today.

Reply:
Expanded the text to differentiate between node/miner, as well as colouring in: https://github.com/filecoin-project/lotus/pull/12364/commits/4d7080a689a7449678b37c5fc1dab80ef5689679